BookingsApi/BookingsApi.DAL/DesignTimeHearingsContextFactory.cs

```diff
@@ -9,9 +9,9 @@ public BookingsDbContext CreateDbContext(string[] args)
         {
             var config = new ConfigurationBuilder()
                 .AddJsonFile("appsettings.json", true)
-                .AddUserSecrets("D76B6EB8-F1A2-4A51-9B8F-21E1B6B81E4F")
                 .AddEnvironmentVariables()
                 .Build();
+            Console.WriteLine(config.GetConnectionString("VhBookings"));
```

Comment:
is this left in by accident?

Reply:
yup, needed to debug why the connection string was missing